src/Avalonia.Controls/SplitButton/ToggleSplitButton.cs

```diff
@@ -36,8 +37,8 @@ public class ToggleSplitButton : SplitButton
         /// Defines the <see cref="IsChecked"/> property.
         /// </summary>
         public static readonly StyledProperty<bool> IsCheckedProperty =
-            AvaloniaProperty.Register<ToggleSplitButton, bool>(
-                nameof(IsChecked));
+            AvaloniaProperty.Register<ToggleSplitButton, bool>(nameof(IsChecked), false,
+                defaultBindingMode: BindingMode.TwoWay);
```

Comment:
nit: I'm a fan of the formatting rule that either has arguments all on one line or all on separate lines (if too long). That matches what was done in the file originally. We have no standards for this though...

Reply:
Sorry, I had copied the line from what was in ToggleButton.

Reply:
Yea, its pretty inconsistent everywhere. Was just noting for this file but if you aren't worried about it, no problem.

It's one of the StyleCop rules too so not as mainstream: https://github.com/DotNetAnalyzers/StyleCopAnalyzers/blob/master/documentation/SA1116.md

Reply:
I updated it to a single line.